src/overlays/actors/ovl_En_Bom/z_en_bom.c

```diff
@@ -668,19 +668,19 @@ void func_808726DC(PlayState* play, Vec3f* arg1, Vec3f* arg2, Vec3f* arg3, s32 a
     f32 temp_f20;
     Vec3f spCC;
     Vec3f spC0;
-    EnBomStruct* ptr = &D_80874650[0];
-    EnBomStruct* ptr2 = &D_80874650[1];
+    PowderKegFuseSegment* ptr = &sPowderKegFuseSegments[0];
+    PowderKegFuseSegment* ptr2 = &sPowderKegFuseSegments[1];
```

Comment:
Can probably just do a quick name on these variables too.

Reply:
To what? We use `ptr` for these sorts of things all the time.

Reply:
Only when we don't know what they are.

Reply:
This isn't true. A collection of stuff I found from the codebase:
En_Tite:
```c
ptr = &this->limbPos[i];
```
Bg_Spdweb:
```c
ptr = &this->collider.elements[i];
```
En_Kame:
```c
ptr = &this->limbPos[5];
```
En_Kusa (admittedly a bit different because it references `unk_194`, but we know that's an array of `EnKusa2`):
```c
        ptr = this->unk_194;
        actor = (EnKusa2*)Actor_SpawnAsChildAndCutscene(
            &play->actorCtx, play, ACTOR_EN_KUSA2, this->actor.world.pos.x, this->actor.world.pos.y,
            this->actor.world.pos.z, 0, Rand_Next() >> 0x10, 0, 1, this->actor.cutscene, this->actor.unk20, NULL);
        *ptr = actor;
```

Reply:
But to cut to the point, I don't know why there are two pointers here. And given that, I'd rather just keep the names we have, because something like `segmentPtr` and `segmentPtr2` don't really seem better than the current names.

Reply:
Slept on this one and I changed my mind on it. I swapped them all over to `fuseSegmentPtr`. Wish I knew why they frequently have two pointers but oh well.